### PR TITLE
Questions page

### LINKED ISSUE #17: 1. A. Show questions
- [x] style the container that contains the questions.
   - [x] container centrally aligned
   - [x] text-align: left
  
This user story is developed on the branch `QuestionsPage`.
This branch is merged into the `master` branch after completion.

### LINKED ISSUE #21: 1. B. Select answer
- [x] style the container with the answers.
- **div**
- [x] centrally aligned.
- **paragraph**
- [x] text-align: left 
- [x] padding 
- [x] hover effects when the user selects the answer
    - [x] border
    - [x] change the background-color to light grey
    - [x] margin

This user story is developed on the branch `QuestionsPage`.
This branch is merged into the `master` branch after completion.

### PR DESCRIPTION
**Pull request by JTLiberona**
CSS Properties Added.

### CODE REVIEW

File: client/styles/index.css [hunk 1]
```diff
@@ -85,3 +85,39 @@ body {
...
+/* Properties for the question container */
+.question-container {
+  background-color: #fff;
+  max-width: 700px;
+  margin: 40px auto;
+  padding: 30px;
+  text-align: left;
+  border-radius: 10px;
+}
+/* Properties for the answers's container */
```
Comment (Moamin-AbuEwaida) [inline client/styles/index.css]: I think you have the same lint:css errors related to spacing between styling tags @JTLiberona

File: client/styles/index.css [hunk 2]
```diff
@@ -85,3 +85,39 @@ body {
...
+.answers-container {
+  background-color: transparent;
+  max-width: 740px;
+  margin: 40px auto;
+  padding: 20px;
+  border-radius: 10px;
+
```
Comment (Moamin-AbuEwaida) [inline client/styles/index.css]: this space is not needed

**Review by Moamin-AbuEwaida — APPROVED** [review]
LGTM

File: client/styles/index.css (hunk 1 above)
Comment (JTLiberona) [reply client/styles/index.css]: Fixed :D

File: client/styles/index.css (hunk 2 above)
Comment (JTLiberona) [reply client/styles/index.css]: Thanks mate! I didn't realized :D

**Review by rayanejsilva — APPROVED** [review]

**Pull request merged**